backend/src/org/commcare/suite/model/Detail.java

```diff
@@ -65,6 +65,10 @@
     // Force the activity that is showing this detail to show itself in landscape view only
     private boolean forceLandscapeView;
 
+    // If this detail is being shown in an entity list, set focus to the LAST item in the list
+    // when it is loaded, instead of the first
+    private boolean focusToBottomOfEntityList;
```

Comment:
Hm, it seems like I should be doing that for all of the custom fields I added here recently, but they have definitely been working without that, which is odd